src/lao.c

```diff
@@ -7,22 +7,25 @@
 #include "ao/ao.h"
 
 /* -- Library Setup/Teardown -- */
+static int has_initialized = 0;
```

Comment:
Not sure how to address this without either introducing pthreads or relying on C11 for `atomic_flag_test_and_set`